docs/iam-structure.md

```diff
@@ -0,0 +1,64 @@
+# IAM Structure
+
+This document aims to outline each of the links in the chain of IAM roles that
+are required for the testing infrastructure. We aim to only use short term
+credentials, and reduce all policies to the minimum required for any given task.
+All of the source code for deploying these roles is in the `infra/` directory.
+
+## Overview
+
+Each of the Prow jobs will first be mounted to a service account. Through
+[IRSA][irsa], this SA will be backed by an IAM role and the pod will
+subsequently be provided credentials. Next, the job entrypoint will look up an
+[SSM parameter][ssm] with a path containing the given service. Assuming that
+exists, it will then assume the role given as the value in that parameter. From
+then on any actions will be authenticated against the service team role.
+
+[irsa]: https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html
+
+<img src="./images/iam-workflow.svg">
+
+## Service Account
+
+We currently maintain two sets of service accounts, each backed by their own IAM
```

Comment:
s/service account/Service Account/g

Also, I would change the phrase "each backed by their own IAM role" to "each associated with their own IAM role"